### PR TITLE
Adds useBulkMutation hook

### LINKED ISSUE #72: useBulkMutate hook
Provide useBulkMutate hook to allow define multiple actions, e.g. if calling a bulk edits or deletes on a single hook.

```
const {mutate} = useMultipleMutate(createAction);

const handleSubmit = (values) => {
    const response = await mutate(values); // values is (e.g.) an array of ids
}
```

### PR DESCRIPTION
**Pull request by josejulio**
Fixes #72

### CODE REVIEW

**Comment by josejulio** [comment]
Finally it seems to be green, ready for a review @marcin-piela

File: src/hooks/useBulkMutation/useBulkMutation.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import { useCallback, useContext, useEffect, useRef, useState } from 'react';
+
+import { Action, QueryResponse, UseBulkMutationResponse } from '../../client/client.types';
+import { QueryError } from '../../client/errors/QueryError';
+import { ClientContext } from '../../context/clientContext/clientContext';
+import { ClientContextType } from '../../context/clientContext/clientContext.types';
+
+type ActionCreator<S, R> = (action: S) => Action<R>;
```
Comment (marcin-piela) [inline src/hooks/useBulkMutation/useBulkMutation.ts]: it should be 
```ts
type ActionCreator<S, R, T> = (action: S) => Action<T, R>;
```

File: src/hooks/useBulkMutation/useBulkMutation.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+type UseBulkMutationState<T> = {
+  responses: Array<QueryResponse<T> | undefined>;
+  loading: boolean;
+};
+
+export const useBulkMutation = <T = any, R = {}, S = any>(
+  actionCreator: ActionCreator<S, R>,
```
Comment (marcin-piela) [inline src/hooks/useBulkMutation/useBulkMutation.ts]:
```ts
  actionCreator: ActionCreator<S, R, T>,
```
I've changed Action typings 2 days ago, that's why :)

Comment (josejulio) [reply src/hooks/useBulkMutation/useBulkMutation.ts]: Missed that, thanks!

**Comment by josejulio** [comment]
@marcin-piela done!

**Pull request merged**